test/utils/ApiClient.spec.js

```diff
@@ -489,5 +492,46 @@ describe("utils/ApiClient", () => {
             });
             callback(new Error(), rawRes);
         });
+
+        it("should call callback with 'body' and 'res' and application/jose+json Content-Type", (cb) => {
```

Comment:
Thanks for verifying the other content type from here, but please do too in Java

Reply:
Ok